app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,116 @@
+require './config/environment'
+require 'rack-flash'
+
+class TweetsController < ApplicationController 
+
+  use Rack::Flash
+  configure do
+    set :public_folder, 'public'
+    set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
+  end
+
+  # the tweets index page
+  # this appears to be the users homepage once they're signed in
+  # can grab a users specific tweets - a user has many tweets
+  get '/tweets' do 
+
+      if current_user
+        @user = current_user
+        erb :'tweets/tweets'
+      else
+        flash[:message] = "You must be logged in"
+        redirect to '/login'
+      end
+  end
+
+   get '/tweets/new' do 
+    @user = current_user
+
+    if @user
+      erb :'/tweets/create_tweet'
+    else
+      flash[:message] = "You must login"
+      redirect '/login'
+    end
+  end
+
+  post '/tweets/new' do
+    @user = current_user
+
+    if params[:content].empty?
+      flash[:message] = "Cannot create an empty tweet!"
+      redirect '/tweets/new'
+    else
+      @tweet = Tweet.new(content: params[:content], user_id: @user.id)
```

Comment:
for lines 46-29

The best way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.